compliance_checker/cf/cf_1_6.py

```diff
@@ -936,13 +938,23 @@ def check_standard_name(self, ds):
                         name
                     ),
                 )
-                if isinstance(standard_name, str):
-                    valid_std_name.assert_true(
-                        standard_name in self._std_names,
-                        "standard_name {} is not defined in Standard Name Table v{}".format(
-                            standard_name or "undefined", self._std_names._version
-                        ),
-                    )
+                if (isinstance(standard_name, str) and
+                    standard_name not in self._std_names):
+                    valid_std_name.out_of += 1
+                    err_msg = (
+                        "standard_name {} is not defined in Standard Name Table v{}."
+                        .format(
+                            standard_name or "undefined",
+                            self._std_names._version))
+                    close_matches = difflib.get_close_matches(standard_name,
+                                                              self._std_names)
+                    if close_matches:
+                        if len(close_matches) == 1:
+                            err_msg += f" Possible close match: {close_matches}"
+                        else:
+                            err_msg += f" Possible close matches: {close_matches}"
```

Comment:
Why not `f" Possible close match(es):`? I know, not as elegant to the user but a few less complexity in the code.

Reply:
Was considering that, I'll revise.

Reply:
No strong opinion on my side BTW. My user brain wants the original commit, my programmer brain wants the change.